lib/workers/repository/update/pr/changelog/source-github.ts

```diff
@@ -54,6 +55,13 @@ export async function getChangeLogJSON({
   // istanbul ignore if
   if (!config.token) {
     if (host.endsWith('github.com')) {
```

Comment:
There's a warning about incomplete URL substring sanitization, a _high_ level warning. Are you sure you actually want to merge this?

Reply:
Yes, it's not a security issue

Reply:
Shouldn't this always be `api.github.com` ?